### PR TITLE
Update block_resize test

### PR DESCRIPTION
**Pull request by suqinhuang**
1. add step description
2. add driver verifiver
3. update disk format

Signed-off-by: Suqin Huang <shuang@redhat.com>

### CODE REVIEW

**Comment by suqinhuang** [comment]
id: 1324369 1324368 1231036 1231035

**Comment by suqinhuang** [comment]
PASS smp_1.1024m.repeat1.run_test.Host_RHEL.m7.u2.qcow2.virtio_blk.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.block_resize

PASS smp_1.1024m.repeat1.run_test.Host_RHEL.m7.u2.qcow2.virtio_blk.up.virtio_net.Win7.x86_64.sp1.io-github-autotest-qemu.block_resize

**Comment by CongSmile** [comment]
How about system disk?

**Comment by suqinhuang** [comment]
discuss with wyu, want auto system disk, case will be separate to system disk and data disk

1. some cfg will be update after extend/shrink system disk, which may cause other case fail
2. could not shrink disk to original one when test with qcow2, so test will fail if need to repeat the test several times

File: qemu/tests/block_resize.py [hunk 1]
```diff
@@ -1,14 +1,16 @@
 import logging
 import re
 
-from autotest.client.shared import error
```
Comment (CongSmile) [inline qemu/tests/block_resize.py]: We need consider compatibility problem, you can refer:
https://github.com/autotest/tp-qemu/commit/6f7b59fa5b6f4dbec4af4c2d2e20d31848d684a4

**Comment by CongSmile** [comment]
Test failed:
14:11:22 ERROR| ShellStatusError: Could not get exit status of command: 'echo list disk > disk &&echo select disk 1 >> disk && echo create partition primary >> disk && echo exit>> disk && diskpart /s disk&& del /f disk'    (output: '\nMicrosoft DiskPart version 10.0.10240\n\nCopyright (C) 1999-2013 Microsoft Corporation.\nOn computer: DESKTOP-N4PVE3R\n\n  Disk ###  Status         Size     Free     Dyn  Gpt\n  --------  -------------  -------  -------  ---  ---\n  Disk 0    Online           30 GB      0 B         \n  Disk 1    Online           10 GB      0 B         \n\nDisk 1 is now the selected disk.\n\nVirtual Disk Service error:\nThere is not enough usable space for this operation.\n\n\n')

File: qemu/tests/block_resize.py (hunk 1 above)
Comment (suqinhuang) [inline qemu/tests/block_resize.py]: I don't think it's a good solution to handle the different between autotest and avocado, 
as more and more difference between them and we won't develop on autotest framework any more.

we can create a branch to tp-qemu for autotest framework.

cc @xutian

**Comment by suqinhuang** [comment]
need to remove internal cmd, which have been sent
-        guest_prepare_cmd = "C:\tools\diskpart_cmd.py --disk_id=1 --op=mount_disk --timeout=120"

File: qemu/tests/block_resize.py (hunk 1 above)
Comment (PandaWei) [inline qemu/tests/block_resize.py]: Obviously,
We have different opinions about it.
The tp-qemu is still kept in autotest organization.【autotest/tp-qemu】,
and it hasn't been *definitely* declared to shift to avocado now.
Maybe we need a copy of tp-qemu/tp-avocado for avocado to thoroughly solve the problem.

**Comment by pezhang** [comment]
Acked-by: Pei Zhang pezhang@redhat.com

Below all 7 tests get PASS:
smp_4.4096m.repeat1.run_test.Host_RHEL.m7.u2.qcow2.virtio_blk.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.block_resize: PASS (42.14 s)

smp_4.4096m.repeat1.run_test.Host_RHEL.m7.u2.qcow2.virtio_scsi.up.virtio_net.RHEL.7.2.x86_64.io-github-autotest-qemu.block_resize: PASS (46.79 s)

smp_4.4096m.repeat1.run_test.Host_RHEL.m7.u2.qcow2.virtio_scsi.up.virtio_net.Win10.x86_64.io-github-autotest-qemu.block_resize: PASS (304.72 s)

smp_4.4096m.repeat1.run_test.Host_RHEL.m7.u2.qcow2.virtio_blk.up.virtio_net.Win2012.x86_64.io-github-autotest-qemu.block_resize: PASS (147.56 s)

smp_4.4096m.repeat1.run_test.Host_RHEL.m7.u2.qcow2.virtio_scsi.up.virtio_net.Win2012.x86_64.r2.io-github-autotest-qemu.block_resize: PASS (143.19 s)

smp_4.4096m.repeat1.run_test.Host_RHEL.m7.u2.qcow2.virtio_scsi.up.virtio_net.Win8.x86_64.io-github-autotest-qemu.block_resize: PASS (311.72 s)

smp_4.4096m.repeat1.run_test.Host_RHEL.m7.u2.qcow2.virtio_scsi.up.virtio_net.Win8.x86_64.1.io-github-autotest-qemu.block_resize: PASS (165.96 s)

**Comment by CongLi** [comment]
Acked-by: Cong Li <coli@redhat.com>

**Pull request merged**